modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {SMProductName} and Kiali
+
+//Kiali attributes are in the works, and possible {KialiProduct} will refer to Kiali provided by Red Hat, and not Kiali Operator provided by Red Hat.
+//Kiali attributes are outside the scope of this PR.
+//TP1 content. IA influx, likely everything will change for GA.
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali provided by Red Hat is composed of three parts:
+
+* {KialiProduct}
+* Kiali Server
+* {SMPlugin}
+
+Working together, they form the user interface (UI) for {SMProduct}. Kiali provides visibility into your service mesh by showing you the microservices and how they are connected.
+
+Kiali provides observability into the Service Mesh running on {ocp-product-title}. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
```

Comment:
Are these redundant? The first refers to any service mesh; the second is the OCP Service Mesh? Do we need them both?
> Kiali provides visibility into your service mesh
>Kiali provides observability into the Service Mesh running